voltdbclient/node_conn.go

```diff
@@ -80,8 +81,10 @@ type nodeConn struct {
 }
 
 func newNodeConn(ci string) *nodeConn {
+	u, _ := parseURL(ci)
```

Comment:
Check the parse error and return nil and err and let operator handle it and log it.